frontend/layouts/discover-page/helpers.ts

```diff
@@ -1,32 +1,50 @@
 import { useIntl } from 'react-intl';
 
+import { compact } from 'lodash-es';
+
 import { SortingOptionType } from 'components/sorting-buttons';
 
 /** Returns the sorting-by types array with translated labels  */
-export const useSortingByOptions = (): SortingOptionType[] => {
+export const useSortingByOptions = (target?: 'projects' | 'openCalls'): SortingOptionType[] => {
```

Comment:
If you used my suggestion above:
```suggestion
export const useSortingByOptions = (target?: Queries.Project | Queries.OpenCall): SortingOptionType[] => {
```

You'd have to change the keys below as well.